server/services/offender/responses/__tests__/nextVisit.spec.js

```diff
@@ -1,79 +1,66 @@
-const { NextVisit } = require('../nextVisit');
-const {
-  placeholders: { DEFAULT },
-} = require('../../../../utils/enums');
+const nextVisit = require('../nextVisit');
 const visitTypeDisplayText = require('../../../../content/visitType.json');
 
 const VISIT_TYPE = 'SCON';
 
 describe('NextVisit', () => {
   it('Should handle an empty response', () => {
-    const nextVisit = NextVisit.from();
+    const nextVisitData = nextVisit();
 
-    expect(nextVisit.startTime).not.toBeDefined();
-    expect(nextVisit.status).not.toBeDefined();
-    expect(nextVisit.visitorName).not.toBeDefined();
-    expect(nextVisit.visitType).not.toBeDefined();
-
-    const formatted = nextVisit.format();
+    expect(nextVisitData.startTime).not.toBeDefined();
+    expect(nextVisitData.status).not.toBeDefined();
+    expect(nextVisitData.visitorName).not.toBeDefined();
+    expect(nextVisitData.visitType).not.toBeDefined();
+    expect(nextVisitData.hasNextVisit).toStrictEqual(false);
+  });
 
-    expect(formatted).toStrictEqual(
+  it('should handle visit details', () => {
+    const response = [
       {
-        hasNextVisit: false,
-        nextVisit: DEFAULT,
-        nextVisitDate: DEFAULT,
-        nextVisitDay: DEFAULT,
-        visitType: null,
-        visitorName: DEFAULT,
-        startTime: DEFAULT,
-        endTime: DEFAULT,
+        visitDetails: {
+          startTime: '2019-12-07T11:30:30',
+          endTime: '2019-12-07T13:00:30',
+          visitType: VISIT_TYPE,
+        },
+        visitors: [],
       },
-      'Should return a notification when no start time or status is available',
-    );
-  });
-
-  it('should handle an incomplete response', () => {
-    const response = {
-      startTime: '2019-12-07T11:30:30',
-      endTime: '2019-12-07T13:00:30',
-      eventStatus: 'SCH',
-    };
+    ];
 
-    const formatted = NextVisit.from(response).format();
+    const nextVisitData = nextVisit(response);
 
-    expect(formatted).toStrictEqual(
-      {
-        hasNextVisit: true,
-        nextVisit: 'Saturday 7 December',
-        nextVisitDate: '7 December',
-        nextVisitDay: 'Saturday',
-        visitType: null,
-        visitorName: DEFAULT,
-        startTime: '11:30am',
-        endTime: '1:00pm',
-      },
-      'Should handle missing visitor name or type',
-    );
+    expect(nextVisitData).toStrictEqual({
+      hasNextVisit: true,
+      nextVisit: 'Saturday 7 December',
+      nextVisitDate: '7 December',
```

Comment:
wonder if we use all these different versions of the date and time. 

Not as part of this but I'd be tempted to move to just return the date and format as neccessary on the frontend